app/src/main/java/com/dsu/pocapplication/news/ui/login/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.pocapplication.news.ui.login.viewmodel
```

Comment:
there is no requirement of package "viewmodel" inside login

Reply:
done